src/support/components/FeedbackQuestionsOverlay.tsx

```diff
@@ -0,0 +1,105 @@
+import React, {FC, ChangeEvent, useContext, useState} from 'react'
+
+// Components
+import {
+  Button,
+  ButtonType,
+  ComponentColor,
+  ComponentStatus,
+  Form,
+  Overlay,
+  TextArea,
+} from '@influxdata/clockface'
+
+// Contexts
+import {OverlayContext} from 'src/overlays/components/OverlayController'
+
+// Types
+import ErrorBoundary from 'src/shared/components/ErrorBoundary'
+
+interface OwnProps {
+  onClose: () => void
+}
+
+const FeedbackQuestionsOverlay: FC<OwnProps> = () => {
+  const {onClose} = useContext(OverlayContext)
+  const [input, setInput] = useState('')
+
+  const handleSubmit = () => {
+    // handle form submit
+  }
+
+  const handleInputChange = (e: ChangeEvent<HTMLTextAreaElement>) => {
+    setInput(e.target.value)
+  }
+
+  const handleValidation = (value: string): string | null => {
+    if (value.trim() === '') {
+      return 'This field cannot be empty'
+    }
+
+    if (value.length >= 2500) {
+      return 'Must be 2500 characters or less'
```

Comment:
I think very technically, this should be:

>Must be 2500 characters or fewer

but I don't think we care that much. https://www.merriam-webster.com/words-at-play/fewer-vs-less

Reply:
haha! "very technically"